Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -45,13 +48,33 @@
     'pinned': 5,
     'userManagement': 6,
     'image': 7,
-    'plagroundError': 8,
```

Comment:
We should keep it there if we are deprecating it.
The new version will be without it